browser/scripts/player.js

```diff
@@ -247,6 +247,10 @@ function CreatePlayer(cb) {
 	E2.core = new Core()
 	
 	E2.dom.webgl_canvas = $('#webgl-canvas')
+	if (E2.dom.webgl_canvas.length < 1)
```

Comment:
@kschzt  this one is band-aid for the build. i've filed issue #1785 which needs a "proper" fix. The player does not need creating on pages that have no canvas to render in. Does the core still need initialising if no player? I'm not sure.